types/setting.go

```diff
@@ -1437,54 +1437,55 @@ func getDefaultSettingFromYAML(defaultSettingYAMLData []byte) (map[string]string
 	return defaultSettings, nil
 }
 
-func ValidateAndUnmarshalToleration(s string) (*v1.Toleration, error) {
-	toleration := &v1.Toleration{}
-
-	// The schema should be `key=value:effect` or `key:effect`
-	s = strings.Trim(s, " ")
-	parts := strings.Split(s, ":")
-	if len(parts) != 2 {
-		return nil, fmt.Errorf("invalid toleration setting %v: should contain both effect and key/value pair", s)
-	}
+func UnmarshalTolerations(tolerationSetting string) ([]v1.Toleration, error) {
+	taintToleration := []v1.Toleration{}
```

Comment:
Suggest renaming `taintToleration` to `tolerations`. Context is already clear, so prefix taint could be optional.